extensions/wikia/VideoHandlers/tests/VideoFileUploaderTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Class VideoFileUploaderTest
+ *
+ * @group MediaFeatures
+ */
+class VideoFileUploaderTest extends WikiaBaseTest {
+
+	/**
+	 * @dataProvider destinationTitleDataProvider
+	 */
+	public function testGetNormalizedDestinationTitle( $areIdentical, $title, $expectedTitle ) {
+		if ( $areIdentical ) {
```

Comment:
It does look like a trivial test, but I want to ensure that the strings are in fact the same, and catch to early if there was a typo in there causing a "false" mismatch.